cmd/internal/base/base.go

```diff
@@ -65,12 +65,15 @@ func (IDType) String() string {
 // InitCmd returns the init command for ent/c packages.
 func InitCmd() *cobra.Command {
 	var target string
+	var newSchemaTemplate string
 	cmd := &cobra.Command{
 		Use:   "init [flags] [schemas]",
 		Short: "initialize an environment with zero or more schemas",
 		Example: examples(
 			"ent init Example",
 			"ent init --target entv1/schema User Group",
+			"ent init --template '<template string>'",
+			"ent init --template file=path/to/tmpl.tmpl",
```

Comment:
```suggestion
			"ent init --template file=path/to/file.tmpl User",
```

Reply:
Please, let's not support string template atm. It's not supported by `ent generate` and I don't see a use case for it.

Reply:
thought I saw it elsewhere (?)

took a second look and couldn't find, will remove